samples/net/wifi/overlay-nrf70.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_WIFI_NRF70=y
+CONFIG_WIFI_NM_WPA_SUPPLICANT=y
+CONFIG_NET_L2_ETHERNET=y
+# Adjust memories as per nRF70 + sample configuration
+CONFIG_MAIN_STACK_SIZE=5200
+CONFIG_SHELL_STACK_SIZE=5200
```

Comment:
These two should be added to the boards folder as they depend on the actual board used (SoC) rather than the sample itself. Lets add board overlays for the nrf DKs :)

Reply:
I agree :) The board `.overlay` and `.conf` contain all of this information. We need to add a set of tested combinations of board/shield to the boards folder, with optimized HEAP usage. I will spend some time working with different combinations of boards to make sure they build and run perfectly with the sample.

The user should only need to do a `west build -p -b <board>` and just have it work, no extra overlays or arguments. This should be possible with board variants, I will look into this now :)

Reply:
> The user should only need to do a west build -p -b <board> and just have it work, no extra overlays or arguments. 

We cannot move the `overlay-nrf70.conf` as default config for the board, because nRF70 driver works with
* WPA supplicant for full Wi-Fi
* no WPA supplicant for scan_only, Raw modes etc

Hence it was moved to sample.

> This should be possible with board variants, I will look into this now :)

You mean adding board variants for all these combinations, it would be too many :)